src/methods/mod.rs

```diff
@@ -1,7 +1,7 @@
 //! This module contains all the RPC methods.
 use std::io;
 
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
```

Comment:
Looks redundant

Reply:
This is used through wildcard imports in all method modules. I’d refactor it, but we might deprecate this implementation in favor of the generated client, so I decided to make the minimal changes

Reply:
Make sense. Thank you